client/src/screens/SignInScreen.tsx

```diff
@@ -0,0 +1,91 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useContext, useState } from "react";
+import { Image, StyleSheet, Text, TextInput, TouchableOpacity, View, } from "react-native";
+import AuthService from "../services/auth.service";
+import { IContextType, SessionContext } from "../services/session-context.service";
+import type { StackScreenProps } from '@react-navigation/stack';
+import { RootStackParamList } from "../utils/navigation-stack";
+
+type SignInScreenProps = StackScreenProps<RootStackParamList>;
+
+export const SignInScreen = ({ navigation } : SignInScreenProps) => {
+    const context = useContext<IContextType | null>(SessionContext);
+
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    const onSignInPress = async () => {
+        try {
+            const userSession = await AuthService.signIn(email, password);
+            if (context !== null) context.updateSession(userSession);
+        } catch (err) {
+            console.error(JSON.stringify(err));
+        }
+    };
+
+    return (<View style={styles.container}>
+        <Image style={styles.image} source={require("../../assets/logo-spic.png")} />
+        <StatusBar style="auto" />
+        <View style={styles.inputView}>
+            <TextInput
```

Comment:
All `TextInput` placeholder value is removed when changing orientation & not focused